### PR TITLE
Make sure that we set None to false for selected and framework

### PR DESCRIPTION
**Pull request by jsancio**

### CODE REVIEW

File: cosmos-model/src/main/scala/com/mesosphere/universe/v3/model/PackageDefinition.scala [hunk 1]
```diff
@@ -59,8 +59,8 @@ object PackageDefinition {
     a: (String, Version, ReleaseVersion),
     b: (String, Version, ReleaseVersion)
   ): Int = {
-    val (aName, aVersion, aReleaseVersion) = a
-    val (bName, bVersion, bReleaseVersion) = b
+    val (aName, _, aReleaseVersion) = a
+    val (bName, _, bReleaseVersion) = b
```
Comment (jsancio) [inline cosmos-model/src/main/scala/com/mesosphere/universe/v3/model/PackageDefinition.scala]: Unused variable.

File: cosmos-render/src/main/scala/com/mesosphere/cosmos/render/MarathonLabels.scala [hunk 2]
```diff
@@ -41,11 +41,14 @@ final class MarathonLabels(
...
-  def nonOverridableLabelsJson: JsonObject = JsonObject.fromMap(nonOverridableLabels.mapValues(_.asJson))
+  def nonOverridableLabelsJson: JsonObject =
+    JsonObject.fromMap(nonOverridableLabels.mapValues(_.asJson))
 
   def nonOverridableLabels: Map[String, String] = {
     Seq(
-      commandJson.map(command => MarathonApp.commandLabel -> MarathonLabels.encodeForLabel(command))
+      commandJson.map(
+        command => MarathonApp.commandLabel -> MarathonLabels.encodeForLabel(command)
+      )
```
Comment (jsancio) [inline cosmos-render/src/main/scala/com/mesosphere/cosmos/render/MarathonLabels.scala]: Formatting change here and above. Lines were too long.

File: cosmos-render/src/test/scala/com/mesosphere/cosmos/render/MarathonLabelsSpec.scala [hunk 3]
```diff
@@ -79,10 +79,10 @@ final class MarathonLabelsSpec extends FreeSpec {
     assertResult(original.maintainer)(result.maintainer)
     assertResult(original.description)(result.description)
     assertResult(original.tags)(resultTags)
-    assertResult(original.selected)(result.selected)
+    assertResult(original.selected.getOrElse(false))(result.selected.get)
     assertResult(original.scm)(result.scm)
     assertResult(original.website)(result.website)
-    assertResult(original.framework)(result.framework)
+    assertResult(original.framework.getOrElse(false))(result.framework.get)
```
Comment (jsancio) [inline cosmos-render/src/test/scala/com/mesosphere/cosmos/render/MarathonLabelsSpec.scala]: We did have a test for this!

File: cosmos-model/src/main/scala/com/mesosphere/cosmos/label/v1/model/PackageMetadata.scala [hunk 4]
```diff
@@ -12,7 +12,7 @@ case class PackageMetadata(
   maintainer: String,
   description: String,
   tags: List[String] = Nil,
-  selected: Option[Boolean] = None,
+  selected: Option[Boolean] = Some(false),
   scm: Option[String] = None,
   website: Option[String] = None,
   framework: Option[Boolean] = None,
```
Comment (cruhland) [inline cosmos-model/src/main/scala/com/mesosphere/cosmos/label/v1/model/PackageMetadata.scala]: Also, we might want a comment here to explain why these fields have `Option[Boolean]` type while `DescribeResponse` uses just `Boolean`.

File: cosmos-render/src/test/scala/com/mesosphere/cosmos/render/MarathonLabelsSpec.scala (hunk 3 above)
Comment (cruhland) [inline cosmos-render/src/test/scala/com/mesosphere/cosmos/render/MarathonLabelsSpec.scala]: This is a good alternative place for the `Option[Boolean]` vs. `Boolean` comment. We just need to put it in a place that will remind us that `Option[Boolean]` is used for a reason.

File: cosmos-model/src/main/scala/com/mesosphere/cosmos/label/v1/model/PackageMetadata.scala (hunk 4 above)
Comment (jsancio) [inline cosmos-model/src/main/scala/com/mesosphere/cosmos/label/v1/model/PackageMetadata.scala]: Done. Thanks!

File: cosmos-render/src/test/scala/com/mesosphere/cosmos/render/MarathonLabelsSpec.scala (hunk 3 above)
Comment (jsancio) [inline cosmos-render/src/test/scala/com/mesosphere/cosmos/render/MarathonLabelsSpec.scala]: I put it in the type `PackageMetadata`. Easier to document there.

**Pull request merged**